src/test/java/com/google/collegeplanner/TestCourseList.java

```diff
@@ -0,0 +1,48 @@
+package com.google.collegeplanner;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.google.collegeplanner.servlets.CourseListServlet;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Tests CourseListServlet */
+@RunWith(JUnit4.class)
+public final class TestCourseList {
+  @Test
+  public void servletResponseHasCourses() {
```

Comment:
`public void servletResponseHasCourses() throws Exception`. Then you don't need the try/catch.